dist/src/main/arctic-bin/conf/config.yaml

```diff
@@ -62,7 +62,7 @@ ams:
 
   terminal:
     backend: local
-    local.spark.sql.session.timeZone: UTC
+  #  local.spark.sql.session.timeZone: UTC
```

Comment:
Remove this line directly?

Reply:
how about removing this line instead of commenting on it?

Besides, we may need to change the configuration in `docs/admin-guides/deployment.md` too.

Reply:
done